### PR TITLE
Feedback about user deleting.

### PR DESCRIPTION
**Pull request by Glomberg**

### CODE REVIEW

File: inc/cleantalk-public-integrations.php [hunk 1]
```diff
@@ -1449,7 +1449,7 @@ function ct_registration_errors($errors, $sanitized_user_login = null, $user_ema
         true
     );
     $ct_result        = $base_call_result['ct_result'];
-
+    ct_hash($ct_result->id);
```
Comment (ArtemAnoshin) [inline inc/cleantalk-public-integrations.php:1452]: why call ct_hash if its results are not used further?

File: inc/cleantalk-public-integrations.php [hunk 2]
```diff
@@ -1654,18 +1654,9 @@ function ct_check_registration_erros($errors, $_sanitized_user_login = null, $_u
...
-        update_user_meta($user_id, 'ct_hash', $apbct_cookie_request_id);
-
-        return;
-    }
-
-    if ( isset($_COOKIE[$apbct_cookie_request_id_label]) ) {
-        if ( update_user_meta($user_id, 'ct_hash', $_COOKIE[$apbct_cookie_request_id_label]) ) {
-            Cookie::set($apbct_cookie_request_id_label, '0', 1, '/');
-        }
+    $hash = ct_hash();
```
Comment (ArtemAnoshin) [inline inc/cleantalk-public-integrations.php:1657]: why call ct_hash with an empty parameter?

**Review by ArtemAnoshin — APPROVED** [review]

File: inc/cleantalk-public-integrations.php (hunk 1 above)
Comment (Glomberg) [reply inc/cleantalk-public-integrations.php:1452]: Using a static variable in the function.

File: inc/cleantalk-public-integrations.php (hunk 2 above)
Comment (Glomberg) [reply inc/cleantalk-public-integrations.php:1657]: Using a static variable in the function.

**Pull request merged**